react/features/base/settings/functions.any.js

```diff
@@ -265,6 +265,12 @@ export function shouldHideShareAudioHelper(state: Object): boolean {
  * @returns {boolean}
  */
 export function shouldHideSelfView(state: Object) {
+    const { conference } = state['features/base/conference'];
+
+    if (conference?.isHidden()) {
```

Comment:
How do Jibris currently hide their local view?

Reply:
trough `config.iAmRecorder` and `config.iAmSipGateway`